### ISSUE TITLE
Can't create backups (and can't send messages)

### ISSUE THREAD

**Issue by rradar** [signalapp/Signal-Android#8624]
- [ x] I have searched open and closed issues for duplicates
- [ x] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

I badly try creating an backup of my signal communications because I have the (well known) issue that I can't send any messages anymore.

I had version 4.32.x installed (with an imported backup from another phone) and recently made the jump to 4.33.x as it was finally available as .apk on the official site (and got rick rolled as well).

Sending messages were already stuck in v.4.32.x (also the latest one which should fix the issue) and backups also impossible from that time.

Turning Backup on in the setting works but the system never creates a backup (Last backup: none). I interpret the log that it's waiting for my positive response that I saved my backup password. But I always did it (trying at least 10 times). I also tried everything else which would came to my mind (also for my other problem that I can't send any message anymore) - this includes, several: phone restarts,  force app stops, delete of app cache, reinstalling the update, disable signal service, re-register signal service.

I'm somehow caught now because I want a clean reinstall (for hopefully solving my problem that I can't send message) but for that I need a (working) backup before.

### Steps to reproduce
- open settings
- turn backup on
- tick the check box
- hit the button

**Actual result:** No Backup is taking. Last Backup: None
**Expected result:** Backup taking should start the next minutes, hours, days

### Screenshots

Something went really wrong, and we can't process that file.

### Device info

**Device:** Samung S7
**Android version:** 8.1.0
**Signal version:** 4.33

### Link to debug log

debug log is following (don't have network on my phone right now and copying the debug log (text) isn't working so I just added a screenshot for now)

**Comment by greyson-signal** [comment]
Please try to reproduce the problem and grab a log afterwards. Without a log, I can't do much.

**Comment by rradar** [comment]
Hey @greyson-signal 

thank's for your feedback! Here is my missing screenshot:

![image](https://user-images.githubusercontent.com/34582688/53182129-2e370f80-35f9-11e9-99ab-451ff4fda233.png)

And here is my debug log:

> 15:20 Backup deactivated
> 15:48 Backup activated
> 15:50 No Backup activity, debug log submitted

https://debuglogs.org/e2438f624d15ae088ce3b557ed6008966b7a8385530f969968c0c10440bee449

**Comment by rradar** [comment]
I only can receive text messages now. Can't receive any more pictures... it's getting worse :disappointed:

**Comment by greyson-signal** [comment]
I don't see any indication of any backup activity at all in the log. Can you post of screenshot of what the backup area looks like? Like the bottom of the "Chats and media" setting screen. I guess also verify that signal has storage permissions, but we should ask for them at that point anyway. And verify you're clicking the 'create backup' button. And if you're curious if backups are being created, you can check /sdcard/Signal/Backups`.

Regarding message sending/receiving, there's a slew of problems Lineage and non-gapps users might be having in regards to executing jobs that aren't currently in my control. Trying to allocate time to look into it.

**Comment by rradar** [comment]
update on this issue.

I made a update to the most recent version (4.38.3). Now not only back ups worked again right away (it was still active in the setting and started right away counting up the messages backin' up) but also I am able again to take part in conversations (not receiving messages only and not beeing able to anwser).

I'm not sure if it made a difference that this time I used a google app store wrapper to get the apk and not the signal website release section where I used to download it? Are the releases reproducible builds? Shouldn't they be exactly the same (bit wise)? Or was it really that the update fixed the problem?

I just leave it open for @greyson-signal to close and answer my question above! Thank's!
